added proposal

hafsaJChaudhry opened the pull request:
Could not add paper proposal under /paper in enkeboll's repository. and did not see a file called paper under my repository. So i am creating a pull request to make my own paper branch. thanks

hafsaJChaudhry committed:
added proposal

Could not add paper proposal under /paper in enkeboll's repository. and did not see a file called paper under my repository. So i am creating a pull request to make my own paper branch. thanks

enkeboll reviewed (changes requested):
I'm not quite sure what your thesis here is.  What problem are you planning on talking about?  The fact that there is lots of data in social media?  the ad algorithms?  You should hone this into a very specific, precise problem statement-- as is this is too vague.  Reach out if you need help.

hafsaJChaudhry committed:
added paper outline

hafsaJChaudhry committed:
Merge pull request #7 from hafsaJChaudhry/outline

added paper outline

hafsaJChaudhry committed:
final research paper 

was originally 15mins long read but chopped it down. Because we had to include journal articles, I wrote a section about the cons of my topic (most journal articles I found were talking about the negative effects while almost all web articles I found was talking about what it was + how to make more engaging content) The contrasting take on the subject caused me to cover both
